src/content/docs/accounts/accounts-billing/account-structure/new-relic-account-structure.mdx

```diff
@@ -1,34 +1,43 @@
-* At the top level of a New Relic account hierarchy is the concept of the "organization." Your group of related accounts that share the same billing are inside a single "New Relic organization." 
-* Currently you can't add accounts to your organization on your own. To add accounts, talk to your New Relic account representative. 
-* To learn how you assign users permissions and access to specific accounts, see [Manage users](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts). 
+A New Relic "account" can be considered a workspace. For example, you might have an account for a specific app, or a set of related hosts and services for a specific initiative or project, or you might have an account for a specific team.
 
-Having trouble accessing a specific account or feature? See [Factors affecting access](/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data). 
+Our [Standard edition](https://newrelic.com/pricing) allows for a single account per organization. [Pro and Enterprise editions](https://newrelic.com/pricing) allow for multiple accounts per organizations. Currently you can't add accounts to your organization on your own. To add accounts, talk to your New Relic account representative. 
+
+### How users access accounts [#account-access]
+
+One effect of the organization structure is that you require a separate login for each New Relic organization on the new model. For example, if you're a contractor requiring access to several New Relic organizations that are on the new model, you'll need to sign out of one organization and into the other. If you have multiple logins associated with a single email address, you can [verify your account to see all associated logins](/docs/accounts/accounts-billing/account-setup/troubleshoot-new-relics-password-email-address-login-problems/#verify-email).
+
+Users inside an organization are granted access to specific accounts, as befitting those users’ belonging to specific teams or projects. To manage users’ access to accounts, you create [access grants](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts), which assign a group of users to a specific role on a specific account. For example, you may assign some users the ability to manage other users or manage billing on some accounts, and assign other users as non-admin full users on some accounts, and assign other users as basic users on some accounts. Our user management system allows you to create the user access you need, whether that’s a relatively simple setup with just a few roles across a few accounts, or a complex one with many roles across many accounts. [Learn more about user management.](/docs/accounts/accounts-billing/new-relic-one-user-management/)
```

Comment:
Worth thinking about how to simplify this language I think. Something like: "...specific accounts, depending on their teams or projects."